### PR TITLE
Fix initial Android support

### PR DESCRIPTION
**Pull request by enfipy**
Add force touches.
Fix UI focus system and touch screen system.
Fix `Moved` event.
Fix examples README.
Update `rodio` to `0.13` with Android support.
Get rid of `supported_android_features` as all features are now supported.
Add Android build CI with `cargo-apk`.

Example in `bevy` works well:

</img>

### CODE REVIEW

**Comment by cart** [comment]
I think its worth trying to squeeze this in to the release. testing on all platforms now (to be safe)

**Comment by cart** [comment]
UI touch "kind of" works for me now. If i "press" a button it stays pressed until i touch something else.

**Comment by cart** [comment]
I think its probably good enough for now though.

**Comment by enfipy** [comment]
Yes. It's still a lot to enhance and rework. But at least it works. I even did a paint app

File: examples/README.md [hunk 1]
```diff
@@ -175,42 +190,46 @@ used for the `Makefile`.
...
-    min_sdk_version = 29
-
-    [[package.metadata.android.feature]]
-    name = "android.hardware.vulkan.level"
-    version = "1"
+```toml
+[package.metadata.android]
+build_targets = ["aarch64-linux-android", "armv7-linux-androideabi"]
+target_sdk_version = 29
+min_sdk_version = 16
```
Comment (PrototypeNM1) [inline examples/README.md:220]: min_sdk_version here is different from the one in Cargo.toml

**Pull request merged**